src/SFA.DAS.QnA.Application/Services/NotRequiredProcessor.cs

```diff
@@ -3,23 +3,23 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
-using Newtonsoft.Json.Linq;
+using System.Text.Json.Nodes;
 using SFA.DAS.QnA.Api.Types.Page;
 
 namespace SFA.DAS.QnA.Application.Services
 {
     public class NotRequiredProcessor: INotRequiredProcessor
     {
-        public bool NotRequired(IEnumerable<NotRequiredCondition> notRequiredConditions, JObject applicationData)
+        public bool NotRequired(IEnumerable<NotRequiredCondition> notRequiredConditions, JsonNode applicationData)
         {
             return notRequiredConditions.Any(nrc =>
-                nrc.IsOneOf != null && nrc.IsOneOf.Contains(applicationData[nrc.Field]?.Value<string>()));
+                nrc.IsOneOf != null && nrc.IsOneOf.Contains(applicationData.AsObject()[nrc.Field]?.GetValue<string>()));
```

Comment:
We'll need to come back to this and other cases where JsonObjects are being referenced by index using the square bracket syntax, since we know that works very differently in Text.Json.

Reply:
Just for the record, you've pointed out that there are no null checks:

> There are 3 places in the code that we see this object[property] code and none have null checks. Two are a contains method and one is just setting a property to the value of object[property]

Reply:
can this be resolved?

Reply:
Ok. Where my concerns are addressed I'm happy for someone else to resolve my comment